templates/web/base/header.html

```diff
@@ -17,6 +17,7 @@
         [% END %]
         <meta name="HandHeldFriendly" content="true">
         <meta name="mobileoptimized" content="0">
+        <link rel="stylesheet" href="/vendor/govuk-frontend/forms.css">
```

Comment:
Shouldn't need this here if it's in the other place?  I think you need to add it to the fixmystreet.com/header/css.html override template as well rather than here.